packages/pluggableWidgets/rating-native/typings/RatingProps.d.ts

```diff
@@ -5,18 +5,7 @@
  */
 import { ActionValue, DynamicValue, EditableValue, NativeImage } from "mendix";
 
-export type AnimationEnum =
-    | "pulse"
-    | "bounce"
-    | "flash"
-    | "jello"
-    | "rotate"
-    | "rubberBand"
-    | "shake"
-    | "swing"
-    | "tada"
-    | "wobble"
-    | "none";
+export type AnimationEnum = "pulse" | "bounce" | "flash" | "jello" | "rotate" | "rubberBand" | "shake" | "swing" | "tada" | "wobble" | "none";
```

Comment:
Same

Reply:
This output is from running release. I also ran prettier subsequent to these changes, prettier was happy